GeeksCoreLibrary/Modules/PostalServices/PostNL/Controllers/PostNLController.cs

```diff
@@ -0,0 +1,231 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using GeeksCoreLibrary.Core.Extensions;
+using GeeksCoreLibrary.Core.Interfaces;
+using GeeksCoreLibrary.Core.Models;
+using GeeksCoreLibrary.Modules.Communication.Interfaces;
+using GeeksCoreLibrary.Modules.GclReplacements.Interfaces;
+using GeeksCoreLibrary.Modules.PostalServices.PostNL.Interfaces;
+using GeeksCoreLibrary.Modules.PostalServices.PostNL.Models;
+using GeeksCoreLibrary.Modules.PostalServices.PostNL.Services;
+
+namespace GeeksCoreLibrary.Modules.PostalServices.PostNL.Controllers
+{
+    [Route("api/post-nl")]
+    public class PostNLController : Controller
+    {
+        private readonly IPostNLService postNlService;
+        private readonly IWiserItemsService wiserItemsService;
+        private readonly ICommunicationsService communicationService;
+        private readonly IStringReplacementsService stringReplacementsService;
+
+        public PostNLController(IPostNLService postNLService, IWiserItemsService wiserItemsService, ICommunicationsService communicationsService, IStringReplacementsService stringReplacementsService)
+        {
+            this.postNlService = postNLService;
+            this.wiserItemsService = wiserItemsService;
+            this.communicationService = communicationsService;
+            this.stringReplacementsService = stringReplacementsService;
+        }
+
+        private async Task<string> GetCountryCode(WiserItemModel orderDetails, string entityName)
```

Comment:
Async functies moeten altijd de suffix `Async` hebben, deze graag hernoemen naar `GetCountryCodeAsync`.

Reply:
Deze functie hoort overigens ook niet thuis in de controller. Graag verplaatsen naar een service.